### PR TITLE
Initial build setup for standalone shim.

### PR DESCRIPTION
**Pull request by amshinde**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: c
+dist: trusty
+complier: gcc
```
Comment (dlespiau) [inline .travis.yml]: typo: "complier"

**Comment by dlespiau** [comment]
While we're here, probably a good idea to add a .gitignore?

lgtm (with the typo fixed/removed)

File: .travis.yml (hunk 1 above)
Comment (amshinde) [reply .travis.yml]: fixed

**Comment by amshinde** [comment]
Added building with clang on travis.
Added pullapprove config file.

**Pull request merged**